test/server.test.js

```diff
@@ -146,7 +147,8 @@ describe('Server file', () => {
         .end((error, response) => {
           expect(error).to.be.null;
           expect(response).to.have.status(202);
-          expect(response.body.message).to.equal(successMessage);
+          expect(response.body.message.includes('Edit successful. Station with id of 1 name changed from')).to.equal(true);
+          expect(response.body.message.includes('to Edit Test Station 1.')).to.equal(true);
```

Comment:
Nice use of includes in the test. I didn't know about that `.to.equal.true` bit on the end of it instead of a `deep.includes`